src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/IntegrationTestBase.kt

```diff
@@ -23,8 +24,19 @@ abstract class IntegrationTestBase {
   @Autowired
   lateinit var keyPair: KeyPair
 
+  @Autowired
+  lateinit var wireMockService: WiremockService
+
   @BeforeEach
   fun clearDatabase() {
     actionPlanRepository.deleteAll() // Will also remove all Goals and Steps due to cascade
   }
+
+  @BeforeEach
+  fun resetWireMock() {
+    with(wireMockService) {
+      resetAllStubsAndMappings()
+      stubHmppsAuthUserMe()
```

Comment:
I've added the stub for the HMPPS Auth 'me' endpoint in the before each, so that all integration tests automatically get it (they will need it - even the `GET` requests)